src/proto/streams/streams.rs

```diff
@@ -1270,6 +1270,7 @@ impl OpaqueStreamRef {
         me.actions.recv.poll_response(cx, &mut stream)
     }
     /// Called by a client to check for a pushed request.
+    #[allow(clippy::type_complexity)]
```

Comment:
can prob just add this to the root... I don't really like this lint tbh

Reply:
will do